src/amber/cli/commands/new.cr

```diff
@@ -10,6 +10,7 @@ module Amber::CLI
         string "-d", desc: "database", any_of: %w(pg mysql sqlite), default: "pg"
         string "-t", desc: "template language", any_of: %w(slang ecr), default: "slang"
         string "-m", desc: "model type", any_of: %w(granite crecto), default: "granite"
+        string "-r", desc: "recipe", default: "none"
```

Comment:
I think if this is just `string "-r", desc: "recipe"` (with no default), the default will be nil and the logic in several places can be simplified.